identify/extensions.py

```diff
@@ -241,6 +241,7 @@
     'CMakeLists.txt': EXTENSIONS['cmake'],
     'COPYING': EXTENSIONS['txt'],
     'Dockerfile': {'text', 'dockerfile'},
+    'flake8': EXTENSIONS['ini'] | {'flake8'},
```

Comment:
a file named `flake8` isn't a valid flake8 configuration

Reply:
Did I misinterpret [this section](https://flake8.pycqa.org/en/latest/user/configuration.html#user-configuration)?

```Note that Flake8 looks for ~\.flake8 on Windows and ~/.config/flake8 on Linux and other Unix systems.```

Or should it not be included as it is not a typical repo file?

Reply:
yeah those aren't going to be checked in -- also I plan to remove the user configuration at some point -- it causes significant confusion and almost nobody uses it afaik

Reply:
I see. I'll remove the commit and force push to my branch.

Reply:
Or can you as collaborator drop the commit in the pull request?

Reply:
I could, but it's easier if you do it!

Reply:
Alright!